docs/05-technical-reference/00-custom-resources/evnt-02-eventingbackend.md

```diff
@@ -0,0 +1,41 @@
+---
+title: EventingBackend
+---
+
+The EventingBackend custom resource definition (CRD) is used to show the current status of the Eventing backend. To get the up-to-date CRD and see the output in the YAML format, run this command:
+
+```shell
+kubectl get crd eventingbackends.eventing.kyma-project.io -o yaml
+```
```

Comment:
Currently we only have / support a single instance of the eventing backend custom resource. We have to make this more clear.
```suggestion
The eventing-backend custom resource is used to show the current status of the Eventing Backend. To get the up-to-date CR and see the output in the YAML format, run this command:

```shell
kubectl get crd eventingbackends.eventing.kyma-project.io eventing-backend -n kyma-system -o yaml
```
```

Reply:
Makes sense!

Reply:
I get this error though when I run the suggested command:
```
Error from server (NotFound): customresourcedefinitions.apiextensions.k8s.io "eventing-backend" not found
```

I think the command should be:
```
kubectl get eventingbackends.eventing.kyma-project.io eventing-backend -n kyma-system -o yaml
```